TestControllers/TestController.psm1

```diff
@@ -407,7 +407,7 @@ Class TestController
 		return $currentTestResult
 	}
 
-	[object] RunTestCase($VmData, $CurrentTestData, $ExecutionCount, $SetupTypeData, $ApplyCheckpoint) {
+	[object] RunOneTestCase($VmData, $CurrentTestData, $ExecutionCount, $SetupTypeData, $ApplyCheckpoint) {
```

Comment:
In previous implementation, there are many functions with different purposes but have similar function name. For example, "RunTestCase", "RunTest", etc.
Here, those confusing function names are renamed with more specific names.